GOVERNANCE.md

```diff
@@ -0,0 +1,129 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
+* Contribution policy for the Release repository.
+* Conduct guidelines for the Working group.
+
+For the current list of WG members, see the project README.md.
+
+## Collaborators
+
+The Release GitHub repository is maintained by the WG (all WG
+members are Collaborators for the Release respository) and additional
+Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These individuals
+are identified by the WG and their addition as Collaborators is discussed
+during the weekly WG meeting.
+
+**Note**: If you make a significant contribution and are not considered for
+commit-access, log an issue or contact a WG member directly and it will
+be brought up in the next WG meeting.
+
+Modifications of the contents of the Release repository are made 
+on a collaborative basis.  Anybody with a GitHub account may propose a
```

Comment:
There are two spaces before `Anybody`, whereas elsewhere you use a single space.